src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -135,6 +135,10 @@ class ReimbursementAccountPage extends React.Component {
             }
         }
 
+        const error = lodashGet(this.props, 'reimbursementAccount.error');
+        const failedValidationAttempts = lodashGet(this.props, 'reimbursementAccount.failedValidationAttempts');
+        const maxAttemptsReached = failedValidationAttempts > CONST.BANK_ACCOUNT.VERIFICATION_MAX_ATTEMPTS;
```

Comment:
Since we don't use `failedValidationAttempts` for anything else could we maybe move this to `fetchFreePlanVerifiedBankAccount()` and set a key of `maxAttemptsReached` instead?